net-analyzer/zeek/zeek-3.0.1.ebuild

```diff
@@ -0,0 +1,125 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit cmake-utils python-single-r1
```

Comment:
Most of the packages are migrating to cmake eclass, cmake-utils will be deprecated

Reply:
Are there docs somewhere? https://devmanual.gentoo.org/eclass-reference/index.html does not mention cmake.eclass :(

Reply:
Nevermind, it's actually a drop in replacement. Switched to using that.

Reply:
Yeah, should be pretty straight forward or checking some recent commits where @a17r replaced cmake-utils with cmake.

Reply:
Yup, everything works after straightforward replacement